outputs/mattermost.go

```diff
@@ -44,6 +44,11 @@ func newMattermostPayload(falcopayload types.FalcoPayload, config *types.Configu
 		field.Short = false
 		field.Value = falcopayload.Time.String()
 		fields = append(fields, field)
+
+		attachment.Footer = "https://github.com/falcosecurity/falcosidekick"
+		if config.Slack.Footer != "" {
```

Comment:
We should use a config for `Mattermost` and not `Slack`, what do you think?

Reply:
makes sense, thanks for your review

Reply:
done